nrrd/tests/test_writing.py

```diff
@@ -158,7 +158,7 @@ def test_write_detached_raw_as_nrrd(self):
         output_filename = os.path.join(self.temp_write_dir, 'testfile_detached_raw.nhdr')
         output_data_filename = os.path.join(self.temp_write_dir, 'testfile_detached_raw.nrrd')
 
-        nrrd.write(output_data_filename, self.data_input, {u'encoding': 'raw'}, detached_header=True)
+        nrrd.write(output_data_filename, self.data_input, {u'encoding': 'raw'}, detached_header=True, relative_data_path= False)
```

Comment:
```suggestion
        nrrd.write(output_data_filename, self.data_input, {u'encoding': 'raw'}, detached_header=True, relative_data_path=False)
```

This line is > 120 characters, can you break it at this argument. Repeat for the one other case. Remove the additional space like shown.

Sorry, never explicitly stated that the syntax is to keep lines less than 120 characters, but it's just the standard I use.